document geo+centroid (#1652)
observablehq/plot: docs/transforms/centroid.md

# Centroid transform

> Plot offers two transforms that derive centroids from GeoJSON geometries: [centroid](#centroid-options) and [geoCentroid](#geocentroid-options). These transforms can be used by any mark that accepts **x** and **y** channels. For instance, a [text mark](../marks/text.md) to label the U.S. States.

Reviewer: “For instance, a text mark to label the U.S. States” is a sentence fragment, and also we shouldn’t capitalize the “s” in states. (That only applies to the proper noun United States.)

```suggestion
Plot offers two transforms that derive centroids from GeoJSON geometries: [centroid](#centroid-options) and [geoCentroid](#geocentroid-options). These transforms can be used by any mark that accepts **x** and **y** channels. Below, a [text mark](../marks/text.md) labels the U.S. states.
```
